ui-tests/tst/software/aws/toolkits/jetbrains/uitests/fixtures/FixtureExtensions.kt

```diff
@@ -0,0 +1,27 @@
+// Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package software.aws.toolkits.jetbrains.uitests.fixtures
+
+import com.intellij.remoterobot.fixtures.CommonContainerFixture
+import com.intellij.remoterobot.fixtures.ComponentFixture
+import com.intellij.remoterobot.fixtures.JTextFieldFixture
+import com.intellij.remoterobot.search.locators.byXpath
+import com.intellij.remoterobot.stepsProcessing.step
+import java.time.Duration
+
+fun ComponentFixture.rightClick() = step("Right click") {
+    runJs("robot.rightClick(component);")
+}
+
+fun CommonContainerFixture.pressOk() = findAndClick("//div[@text='OK']")
+fun CommonContainerFixture.pressDelete() = findAndClick("//div[@text='Delete']")
+
+fun CommonContainerFixture.findAndClick(xPath: String) = findByXpath(xPath).click()
```

Comment:
I would mark this and next method with `@Language("XPath")` on the argument

Reply:
Added